classes/footer.php

```diff
@@ -108,13 +108,23 @@ public function get_data(): array {
         $files = $fs->get_area_files($contextid, 'tool_emailtemplate', 'images');
 
         $data['images'] = [];
+
+        $sql = "SELECT timemodified
+                  FROM {config_log}
+                 WHERE plugin = 'tool_emailtemplate' AND name = 'template'
+                 ORDER BY id DESC LIMIT 1";
+        $record = $DB->get_record_sql($sql);
```

Comment:
better to move the limit into the get_record_sql param. You could also just do max(timemodified) 

also best practice is to sort by a date not an id and in some edge cases ids are not in order